xml/http_servers.xml

```diff
@@ -3376,12 +3376,13 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^Boa/(\d+\.[a-zA-Z0-9\.\-]+)">
+   <fingerprint pattern="^Boa/([\d\.]+(?:[a-zA-Z0-9\.\-]*))">
```

Comment:
The reason is the same as it was above -- the `.` in square brackets does not need to be escaped.  I'll fix that because I fixed it in response @pmiseiko-r7's other comment too.  

To the point about not needing to escape the `-` because it is at the end, true, but I'm going to avoid that problem and just change this regex  to simply look for something that begins with a digit and a period and then contains any non-whitespace.